vendor/bat-native-ledger/src/bat/ledger/internal/promotion.cc

```diff
@@ -196,6 +196,12 @@ bool ParseFetchResponse(
         promotion->expires_at = time.ToDoubleT();
       }
 
+      auto* legacyClaimed = item.FindKey("legacyClaimed");
+      if (!legacyClaimed || !legacyClaimed->is_bool()) {
```

Comment:
`is_bool` check not needed

Reply:
Was following existing example, will remove :+1:

Reply:
Done